.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: node_js
+os:
+  - linux
+  - osx
```

Comment:
Although osx is similar to linuxes in most regards and I think we've never ever had an OS X-only issue, I'm still adding this just in case.